dart/lib/src/sentry.dart

```diff
@@ -70,19 +68,11 @@ class Sentry {
         ? const String.fromEnvironment('SENTRY_DSN')
         : options.dsn;
 
-    // we need to execute integrations in a specific order sometimes,
-    // and this initialIntegrations list makes it possible to inject and add
-    // integrations before the runZonedGuardedIntegration gets added
-    if (initialIntegrations != null && initialIntegrations.isNotEmpty) {
-      initialIntegrations
-          .forEach((integration) => options.addIntegration(integration));
-    }
-
     // Throws when running on the browser
     if (!isWeb) {
       // catch any errors that may occur within the entry function, main()
       // in the ‘root zone’ where all Dart programs start
-      options.addIntegration(IsolateErrorIntegration());
+      options.addIntegrationByIndex(0, IsolateErrorIntegration());
```

Comment:
if not adding this method (addIntegrationByIndex), we'd need to refactor more things to set default values via ctor, I still wanna do it but on another PR, because it's not going to be breaking changes anyway